python_coreml_stable_diffusion/attention.py

```diff
@@ -39,7 +52,7 @@ def split_einsum(q, k, v, mask, heads, dim_head):
             attn_weights[head_idx] = attn_weights[head_idx] + mask
 
     attn_weights = [
-        aw.softmax(dim=1) for aw in attn_weights
```

Comment:
Would the softmax be decomposed even if activation quantization is not used? Wouldn't that have an adverse effect on perf (whether the model is running on GPU or NE)?

Reply:
Can we somehow only do this decomposition, when the activation quantization script is invoked?